lib/active_public_resources/oauth/vimeo.rb

```diff
@@ -0,0 +1,36 @@
+require 'httparty'
+require 'base64'
+
+module ActivePublicResources
+  module OAuth
+    class Vimeo
+      attr_reader :token
+
+      AUTHORIZE_URL = 'https://api.vimeo.com/oauth/authorize/client'
+      VERIFY_URL = 'https://api.vimeo.com/oauth/verify'
+
+      def initialize(consumer_key, consumer_secret)
+        @token = Base64.urlsafe_encode64("#{consumer_key}:#{consumer_secret}")
+      end
+
+      def get_access_token
+        response = HTTParty.post(
+          AUTHORIZE_URL,
+          body: { grant_type: 'client_credentials' },
+          headers: { "Authorization" => "Basic #{@token}" }
+        )
+        return response['access_token']
+      end
+
+      def verify_token?(token)
+        response = HTTParty.get(
+          VERIFY_URL,
+          headers: { "Authorization" => "Bearer #{token}" }
+        )
+        return true if response.code == 200
+        return false if response.code == 401
```

Comment:
Maybe we could combine these lines:
```
return true if response.code == 200
return false if response.code == 401
```
into just
```
response.code == 200
```
That way anything but a 200 returns false